docs/application/dotnet/guides/tizen.circularUI/CircleDateTimeSelector.md

```diff
@@ -0,0 +1,97 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector
+summary: CircleDateTimeSelector control guide
+---
+
+# CircleDateTimeSelector
+
+`CircleDateTimeSelector` is a view for Date Picker and Time Picker.
+You can scroll it by [Bezel interaction](https://developer.tizen.org/design/wearable/interaction/bezel-interactions).
+This view is the size that covers the entire screen.
+In order to receive [Rotary event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary), it must be registered as `RotaryFocusObject` property of [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage).
```

Comment:
Register RotaryFocusObject as a property of CirclePage to support the Rotary event.